### ISSUE TITLE
Reproducing the numbers on the synthetic test set

### ISSUE THREAD

**Issue by mtli** [gulvarol/surreal#9]
Hi, I am trying to reproduce the segmentation results on the synthetic test set as said in page 5 of the paper (69.13% IoU, 80.61% accuracy). However, I couldn't match those numbers using either the pre-trained model nor a model trained from scratch, and they are 2~3% lower for both metrics. The only thing I changed from the out-of-the-shelf code is the dataRoot parameters. Could you shed some light on reproducing those numbers?

**Comment by gulvarol** [comment]
Hi, at that time I was working with the .png images which I couldn't release because of space problems. Maybe the difference comes from the compression on the .mp4 files. I haven't checked it myself.

**Comment by gulvarol** [comment]
Sorry for the delay, I did a quick test:

With 8 stacks, on the synthetic test set of mp4 images:
```
    |  mp4 model  | png model
------------------------------
IOU |    66.66    |  61.14
Acc |    77.99    |  72.76
```

So they are indeed 2-3% less than when trained/tested on png images.

**Comment by mtli** [comment]
This is the same with the number I am getting. Thanks for the verification!

**Comment by Dominoer** [comment]
Hi, I am reproducing the depth estimation results on the synthetic test set. But the results I got are quite different from the results reported in the paper. As the paper suggested 'To interpret the error in real world coordinates, we multiply it by the quantization constant (45mm).' I can only get 4.39 * 45mm = 197.8mm (RMSE). That number is far bigger than the number reported in the paper. It would be very appreciated if you could run your released model on the depth estimation baseline too.
